.github/workflows/ci.yml

```
@@ -11,7 +11,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v1
```

```suggestion
        uses: actions/setup-python@v4
```
should get rid of the deprecation warning and the latest for `actions/checkout` is 3.